istioctl/pkg/auth/testdata/validator/notfound-role-in-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "rbac.istio.io/v1alpha1"
```

Comment:
Please add a test case that a file containing ServiceRole after the ServiceRoleBinding. i.e. the binding is referring to a role in the same file.

Reply:
This test has it https://github.com/istio/istio/pull/14058/files#diff-621e614281d0c8cbba912de75055cb52R102